runtime-modules/content-directory/src/permissions.rs

```diff
@@ -122,7 +121,6 @@ impl<T: Trait> Default for Actor<T> {
 }
 
 impl<T: Trait> core::fmt::Debug for Actor<T> {
-    #[cfg(feature = "std")]
     fn fmt(&self, formatter: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
```

Comment:
Why did you remove conditional compilation for this and other Debug implementations>

Reply:
The purpose of this, i still getting errors, related to T: Trait inheritance, when its enabled.

Reply:
Ok. Please, keep in mind to remove those implemented traits after the refactoring.